tests/system/providers/google/cloud/bigquery/example_bigquery_to_postgres.py

```diff
@@ -325,36 +326,42 @@ def setup_connection(ip_address: str) -> None:
         trigger_rule=TriggerRule.ALL_DONE,
     )
 
-    delete_connection = BashOperator(
-        task_id="delete_connection",
-        bash_command=f"airflow connections delete {CONNECTION_ID}",
-        trigger_rule=TriggerRule.ALL_DONE,
-    )
+    @task(task_id="delete_connection")
+    def delete_connection(connection_id: str) -> None:
+        session = Session()
+        log.info("Removing connection %s", connection_id)
+        query = session.query(Connection).filter(Connection.conn_id == connection_id)
+        query.delete()
+        session.commit()
 
-    # TEST SETUP
-    create_bigquery_dataset >> create_bigquery_table >> insert_bigquery_data
-    create_gce_instance >> setup_postgres
-    create_gce_instance >> get_public_ip_task >> setup_connection_task
-    [setup_postgres, setup_connection_task, create_firewall_rule] >> create_pg_table
+    delete_connection_task = delete_connection(connection_id=CONNECTION_ID)
 
     (
-        [insert_bigquery_data, create_pg_table]
+        # TEST SETUP
+        create_gce_instance
+        >> create_bigquery_dataset
+        >> create_bigquery_table
+        >> insert_bigquery_data
+        >> get_public_ip_task
+        >> create_connection_task
+        >> create_firewall_rule
+        >> setup_postgres
+        >> create_pg_table
         # TEST BODY
         >> bigquery_to_postgres
         >> update_pg_table_data
         >> create_unique_index_in_pg_table
-        >> bigquery_to_postgres_upsert
+        # >> bigquery_to_postgres_upsert
```

Comment:
Is this a temporary removal?

Reply:
No, it's here by mistake! I brought it back. Thanks for noticing!